internal/test/consensus/scenarios/fork_and_select_v1/run.sh

```
@@ -0,0 +1,217 @@
+USAGE
+            exit 0;;
+        *) echo "unknown argument: $1" >&2; exit 2;;
+    esac
+done
+
+if [[ -z "${OUT_PATH}" ]]; then
+    OUT_PATH="${GOLDEN_PATH}"
+fi
+mkdir -p "$(dirname "${OUT_PATH}")"
+OUT_DIR="$(cd "$(dirname "${OUT_PATH}")" && pwd)"
+OUT_BASENAME="$(basename "${OUT_PATH}")"
```

Shellcheck (0.11.0)

[warning] OUT_BASENAME appears unused. Verify use (or export if used externally).

(SC2034)